api/src/Controller/StatsController.php

```diff
@@ -78,4 +78,68 @@ public function getAdminUserAccountReportData(Request $request, RestFormatter $f
             'ActivatedAdminAccountsUsedWithin90Days' => $countOfActivatedAdminAccountsUsedWithin90Days,
         ];
     }
+
+    /**
+     * @Route("stats/assets/total_values", methods={"GET"})
+     * @Security("is_granted('ROLE_SUPER_ADMIN')")
+     */
+    public function getAssetsTotalValueData()
+    {
+        $ret = [
+            'lays' => ['liquid' => 0, 'non-liquid' => 0],
+            'profs' => ['liquid' => 0, 'non-liquid' => 0],
+            'pas' => ['liquid' => 0, 'non-liquid' => 0],
+            'grandTotal' => 0,
+        ];
+
+        $lays = $this->reportRepository->getAllSubmittedReportsWithin12Months('LAY');
```

Comment:
What's the indexing like on these tables? Any potential for slowness on the whole data set or impact on other parts of the system?

Reply:
We index on `submit_date`, `report_status_cached` and `submitted`. Querying preprod was pretty instantaneous in Cloud9 so I'm comfortable this shouldn't impact the live service.